connections/src/app/app.module.ts

```diff
@@ -21,6 +21,7 @@ import { QrCodeSaveComponent } from './qr-save-mock/qr-code-save/qr-code-save.co
 import { QrCodeDisplayComponent } from './qr-save-mock/qr-code-display/qr-code-display.component';
 import { MaterialModule } from './material/material.module';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { QrUserProfileComponent } from './qr-user-profile/qr-user-profile.component';
```

Comment:
Do the Module import, also, or rather... instead of just the componet... when you get to Biotool